packages/documentation/pages/usage/components/form-fields.vue

```diff
@@ -1670,7 +1694,50 @@ export default defineComponent({
 			}),
 		)
 
+		watch(
+			() => settings.value.component,
+			(newComponent, oldComponent) => {
+				if (
+					newComponent === 'KtFieldPassword' &&
+					oldComponent !== 'KtFieldPassword'
+				)
+					settings.value = {
+						...settings.value,
+						additionalProps: {
+							...settings.value.additionalProps,
+							autoComplete: 'current-password',
```

Comment:
Isn’t `autoComplete` unsupported by this PR’s `KtFieldPassword` changes?

Reply:
`KtFieldPassword` already supports `autocomplete` but it is restricted to `new-password` and `current-password` ( and now `one-time-code` too).

Other fields support `on`, `off`, `<token>`.

`KtFieldPassword` omits the type from `KottiField.potentiallySupportedPropsSchema` and implements its own.